packages/sdk/src/index.ts

```diff
@@ -56,8 +56,13 @@ export {
   TriggerPatterns,
   TeamsFxAdaptiveCardActionHandler,
   TeamsFxBotCommandHandler,
+  TeamsFxBotSsoCommandHandler,
+  SsoExecutionActivityHandler,
+  SsoConfig,
 } from "./conversation/interface";
 export { ConversationBot } from "./conversation/conversation";
+export { SsoExecutionDialog } from "./conversation/sso/ssoExecutionDialog";
```

Comment:
Why exposing these two class? How will these be used in user code?

Reply:
For SsoExecutionDialog is a normal bot dialog that can be used directly by advance user if he doesn't want to use our ConversionBot.


For DefaultSsoExecutionActivityHandler, it can be used as below, since we will comment the ssoConfig part for the template/sample, I expose this class to let user understand what the default value we're using:
```ts
export const commandBot = new ConversationBot({
  // The bot id and password to create BotFrameworkAdapter.
  // See https://aka.ms/about-bot-adapter to learn more about adapters.
  adapterConfig: {
    appId: process.env.BOT_ID,
    appPassword: process.env.BOT_PASSWORD,
  },

  command: {
    enabled: true,
    commands: [new HelloWorldCommandHandler() ],
    ssoCommands: [new ProfileSsoCommandHandler()],
    // ssoConfig is optional and by default using the values below
    ssoConfig: {
      CustomSsoExecutionActivityHandler: DefaultSsoExecutionActivityHandler,
      scopes: ["User.Read"],
      userState: new UserState(storage),
      conversationState: new ConversationState(storage),
      dedupStorage: storage,
      teamsFxConfig: {
        clientId: process.env.M365_CLIENT_ID,
        clientSecret: process.env.M365_CLIENT_SECRET,
        tenantId: process.env.M365_TENANT_ID,
        authorityHost: process.env.M365_AUTHORITY_HOST,
        initiateLoginEndpoint: process.env.INITIATE_LOGIN_ENDPOINT,
        applicationIdUri: process.env.M365_APPLICATION_ID_URI
      }
    }
  },
```